### PR TITLE
make option to have tracking data to plot

### PR DESCRIPTION
**Pull request by Cryaaa**
For example Compatibility with Ko's dataset restored

### CODE REVIEW

**Comment by Cryaaa** [comment]
Does anybody know why it can't import the tests anymore? the files I changed are minimal.....

**Comment by zoccoler** [comment]
Hi Ryan, 

Could it be because we exhausted the [time limit for tests](https://github.com/organizations/BiAPoL/settings/billing) of the BiA-PoL account for this month?

File: napari_clusters_plotter/_plotter.py [hunk 1]
```diff
@@ -333,6 +335,12 @@ def manual_clustering_method(inside):
         update_button = QPushButton("Update Axes/Clustering Selection Boxes")
         update_container.layout().addWidget(update_button)
 
+        # Update measurements button
```
Comment (lazigu) [inline napari_clusters_plotter/_plotter.py:338]:
```suggestion
        # Tracking data checkbox
```

**Comment by haesleinhuepf** [comment]
I'm curious: Would it also be possible to avoid having that checkbox and instead checking if the `layer.features` do contain a `frame` column or not? I'm asking because if a timelapse dataset is loaded (`len(viewer.dims.current_step) == 4`) an no column 'frame' is present, we could visualize Ko's data, correct?

**Comment by Cryaaa** [comment]
That is true and we could do it that way. On the other hand I’d the column is missing no error will pop up and the cluster image would be wrong. I do like the idea of removing checkboxes though and this was just a quick fix for the deep lineage meeting…

**Comment by haesleinhuepf** [comment]
We could make a warning pop up 🌞

**Comment by Cryaaa** [comment]
Actually, I think a warning everytime the plot is redrawn is annoying. I think we'll do the automatic detection and then make it very clear in the documentation.

**Pull request merged**